File: src/test/java/kitchenpos/fixtures/ProductFixtures.java
package kitchenpos.fixtures;

import kitchenpos.products.tobe.domain.Product;

import java.math.BigDecimal;

public class ProductFixtures {

    public static Product createProduct(final String name, final BigDecimal price) {
        Product product = new Product(name, price);
        return product;
    }

    /**
     * OrderServiceTest
     */
    public static Product product() {
        return product("후라이드치킨", 16_000L);
    }

    public static Product product(final String name, final long price) {
        final Product product = new Product(name, BigDecimal.valueOf(price));
        return product;
    }
}

File: src/test/java/kitchenpos/testsupport/ProductFixtures.kt
package kitchenpos.testsupport

import java.math.BigDecimal
import java.util.UUID
import kitchenpos.products.domain.Product

object ProductFixtures {
    fun createProduct(
        id: UUID? = UUID.randomUUID(),
        name: String? = "test-product-name",
        price: BigDecimal = 1000.toBigDecimal()
    ): Product {
        return Product().apply {
            this.id = id
            this.name = name
            this.price = price
        }
    }
}